Releases/v1.4.0 rc1 (#176)
fyne-io/fyne-cross: CHANGELOG.md

# Changelog - Fyne.io fyne-cross

## Unreleased

## 1.3.0 - 16 Jul 2022

### Added

- Add support for web target #92

> - Add CI job to build calculator app #104

Reviewer: I am not so sure that we want CI stuff to be in the changelog. Not relevant for users

Author: That was actually from the previous release. I have update the changelog with v1.4.0

Author: I don't know what git is doing here, but this was part of the previous release change not this one. It feels incorrect to go back and remove things from the CHANGELOG after it was published.

Reviewer: Ah, I see. Sorry. My GitHub page was messing with me and made it look like part of this release

# Changelog - Fyne.io fyne-cross

## 1.4.0 - 13 Mar 2023

### Added

- Add support for Kubernetes
- Add ability to specify a different registry
- Support for fyne metadata

### Changed

- Pull image from fyne-cross-image repository
- Simplify `fyne-cross darwin-sdk-extract` by getting from specified Apple SDK
the needed file to build darwin application. The file are kept on the host in a

> directory and mounted in the darwin image container for each build

Reviewer: I think we need to improve the wording a bit here. It is a bit unclear.

Reviewer: ```suggestion
- Simplify `fyne-cross darwin-sdk-extract` by getting the needed files from the Apple SDK and then mounting them in the container image for each build
```
